resources/user-agents/browsers/whale/whale.json

```diff
@@ -79,6 +79,33 @@
           ]
         }
       ]
-    }
+    },
+    {
+        "userAgent": "Whale Browser for Android",
+        "browser": "whale browser",
+        "platform": "Android",
+        "engine": "Blink",
+        "device": "general Mobile Device (Touch)",
+        "properties": {
+          "Parent": "DefaultProperties",
+          "Comment": "Whale Browser #MAJORVER#.#MINORVER#",
```

Comment:
Please remove the version tokens (`#MAJORVER#` and `#MINORVER#`) here.

Reply:
Thank you for your review. I have corrected that you pointed.